gems/sorbet/lib/require_everything.rb

```diff
@@ -106,6 +104,27 @@ def self.patch_kernel
 
   private
 
+  def self.rb_file_paths
+    sorbet_exec = File.realpath("#{__dir__}/../bin/srb")
+    output = `#{sorbet_exec} tc -p file-table-json --stop-after=parser`
+    # This returns a hash with structure:
+    # { files:
+    #   [
+    #     {
+    #       "strict": ["Ignore"|"False"|"True"|"Strict"|"Strong"|"Stdlib"],
+    #       "path": "./path/to/file",
+    #       ...
+    #     }
+    #     ...
+    #   ]
+    # }
+    parsed = JSON.parse(output)
+    parsed['files']
+      .reject{|file| ["Ignore", "Stdlib"].include?(file["strict"])}
+      .filter{|file| File.file?(file["path"])} # Some files have https:// paths. We ignore those here.
```

Comment:
Switching from `filter` to `select` should get the snapshot tests passing.

```suggestion
      .select{|file| File.file?(file["path"])} # Some files have https:// paths. We ignore those here.
```

Reply:
:+1: Forget to switch my RVM to `2.4.4` 🙈 Added here https://github.com/sorbet/sorbet/pull/1351/commits/668484c74689a621397bc008e1192ac32694da4d